src/data/cacheUpdates.ts

```diff
@@ -292,7 +292,7 @@ const cacheUpdates = {
             cache.modify({
               id: cache.identify(data.processedColony),
               fields: {
-                canMintNativeToken: () => canMintNativeToken,
+                canUserMintNativeToken: () => canMintNativeToken,
```

Comment:
Shouldn't we add `canColonyMintTokens` here as well?

Reply:
From what I understood when I talked with Raul about the usefulness of adding `canColonyMintTokens` to this file... It seems like this may not be necessary as this file is the old way in which we determine how we updated the cache after we received the results of a query, but we haven't been using/updating it all this time.

It could be argued that it could alleviate the amount of data that the dapp needs to query or receive, but it could also be detrimental for the performance because it's extra stuff that the client needs to do every time, for example, a user reloads the dapp and it's data that we want to stay as fresh as possible anyways.

Due to that, I think I won't add `canColonyMintTokens` yet. Also because it's not as straight-forward to get as `canUserMintNativeToken`